### PR TITLE
Update to RN0.60 and fix compatibility issues

### PR DESCRIPTION
**Pull request by jazibjafri**
- Update docs
- Add types

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,62 @@
...
 [![npm version][npm shield]][npm url]
 
 React Native wrapper for the [Android Activity Recognition API][1] and [CMMotionActivity][3]. It attempts to determine the user activity such as
 driving, walking, running and cycling. Possible detected activities for android are [listed here][2] and for iOS are [listed here][3].<br/>
-Updated January 7th and tested with react-native v0.57.5
 
 [1]: https://developers.google.com/android/reference/com/google/android/gms/location/ActivityRecognition
 [2]: https://developers.google.com/android/reference/com/google/android/gms/location/DetectedActivity
 [3]: https://developer.apple.com/reference/coremotion/cmmotionactivity
 [4]: https://facebook.github.io/react-native/docs/linking-libraries-ios.html#manual-linking
```
Comment (TouqeerH) [inline README.md]: do we really need reference about manual linking? because we don't need it in RN 0.60+

File: README.md [hunk 2]
```diff
@@ -122,30 +116,25 @@ this.unsubscribe()
...
-  });
-
-...
-
-// Start activity detection
-
-const detectionIntervalMillis = 1000
-ActivityRecognition.start(detectionIntervalMillis)
-
+    const interval = 1000;
```
Comment (TouqeerH) [inline README.md]: please define interval unit i.e. ms

File: README.md (hunk 1 above)
Comment (jazibjafri) [reply README.md]: Done

**Review by monisamjad — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jazibjafri) [reply README.md]: Done

**Review by TouqeerH — APPROVED** [review]

**Pull request closed without merging**